### DISCUSSION TITLE
Reverting to standard after sleep/shutdown

### DISCUSSION THREAD

**Discussion by PsychoFox11** [seerge/g-helper#2954] [Q&A] [answered]
I'm using Ghelper 0.181.0 on a ROG Zephyrus M16 GU604VI.
This may be an intended behavior or something on my part, but wanted to ask!

I only recently started using 'eco' mode for the GPU. However, I find when returning to the PC, especially if pressing the power button is needed to wake it, it is back on standard.
I'm using eco (and disabled CPU boost) as it would get REALLY hot when playing a game that doesn't need much power, just by the dGPU being used. It would also still drain the battery when charging on USB-C while playing. Playing with the exact same settings with dGPU disabled, the game runs identically - I see no difference whatsoever (point and click game), but my PC can charge off USB-C and it doesn't make me worry about burns lol.

It also reverts to 240hz instead of 60hz which is all I need currently. It remembers the other settings. Any way to keep Eco/60hz after a shutdown/sleep?

**Comment by seerge** [comment, marked as answer]
@PsychoFox11 hello, if you have `Enable GPU on shutdown` checked under `Extra` then it does exactly that :) https://github.com/seerge/g-helper/discussions/1042

**Reply by PsychoFox11** [reply]
Ah ok, I see that it says it prevents an issue with Eco mode, so I thought it needed to be on, but also expected it to disable it again after startup to return to eco mode after preventing the issue.

**Reply by seerge** [reply]
You can just use `Optimized` it will automatically set back `Standard` when plugged in or `Eco` when on battery

**Reply by PsychoFox11** [reply]
Well that's my issue, I am playing plugged in. Battery drains a bit too fast while gaming otherwise. I just don't want all the heat that comes with Standard when playing a simple game - like I feel this could burn me almost, laptop is pretty new to me but dang it gets hot (and reading online, others agree). I think it will at least stay charged on its own charger, but that doesn't fix the heat, and where I'm using it a USB-C charger is more convenient (it is a macbook charger). I keep it plugged in on eco and it stays at my charging limit, or charges while I'm playing, and the game looks identical without frying anything nearby (same exact settings, but again, it is a point and click adventure, not anything demanding). I did play unplugged accidentally last night on eco (charger was plugged into my PC but not the other end) and after an hour was needing to plug in, when I'm playing several hours at a time.

I do however think you answer my question on the thread of the above issue. Is there an issue with Eco if you don't try to switch to standard though? I'd just like to stay in eco no matter what for a few weeks anyway until I'm done with this game.

**Reply by seerge** [reply]
There is no issue with Eco or Standard itself.  

But there is a (possible, affecting only some devices) hiccup in Nvidia driver, that won't let you control display brightness if you boot in Eco mode first (i..e when nvidia GPU is completely disabled) and then enable it. And a checkbox under `Extra` i mentioned, prevents this from happening.

If you don't suffer from brightness issue at all (or don't care about it), you don't need to check it  under `Extra`. Set Eco and use it all the time. It shouldn't revert to Standard then at all :)
